pkg/driver/docker/docker.go

```diff
@@ -284,10 +284,31 @@ func (c dockerClient) StartBuildkitd(ctx context.Context, tag, name string, bc *
 	}
 	created, _ := c.Exists(ctx, name)
 	if created {
-		err := c.ContainerStart(ctx, name, types.ContainerStartOptions{})
+		status, err := c.GetStatus(ctx, name)
 		if err != nil {
-			return name, err
+			return name, errors.Wrap(err, "failed to get container status")
 		}
+
+		if status == "paused" {
+			logger.Info("container was paused, unpause it now...")
+			_, err := c.ResumeContainer(ctx, name)
+			if err != nil {
+				return name, errors.Wrap(err, "failed to unpause container")
+			}
+		} else if status == "exited" {
```

Comment:
Is it possible to get this status? We already use `AutoRemove`.

Reply:
Just in case. As @gaocegege mentioned in issue #1407 :

> The bootstrap fails if the envd_buildkitd container is exited. We need to figure out how to resume it, or recreate it.